### PR TITLE
Add mRo KitCAN UAVCAN Node+Compass+Baro

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-mro-kitcan.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+.. image:: ../../../images/mro-kitcan.png
+
+Features
+========
+
+- UAVCAN to GPS (UART interface)
+- UAVCAN to I2C Airspeed Sensor (MS5525 default configuration) and/or MAGNETOMETER (autodetect)
+- UAVCAN integrated peripherals:
+   - RM3810 High Precision Magnetometer
```
Comment (pkocmoud) [inline common/source/docs/common-mro-kitcan.rst:18]: PNI RM3100

**Review by pkocmoud — CHANGES_REQUESTED** [review]
The MAG model needs to be corrected, then it looks good to me.

File: common/source/docs/common-mro-kitcan.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-mro-kitcan.rst]: fixed

**Comment by Hwurzburg** [comment]
merging

**Pull request merged**